autoscaler/controllers/gateway/config/quickwit.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	odigosv1 "github.com/keyval-dev/odigos/api/odigos/v1alpha1"
+	commonconf "github.com/keyval-dev/odigos/autoscaler/controllers/common"
+	"github.com/keyval-dev/odigos/common"
+)
+
+const (
+	qwUrlKey = "QUICKWIT_URL"
+)
+
+type Quickwit struct{}
+
+func (e *Quickwit) DestType() common.DestinationType {
+	return common.QuickwitDestinationType
+}
+
+func (e *Quickwit) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) {
+	if url, exists := dest.Spec.Data[qwUrlKey]; exists {
+		exporterName := "otlp/quickwit/" + dest.Name
+
+		currentConfig.Exporters[exporterName] = commonconf.GenericMap{
+			"endpoint": url,
+			"tls": commonconf.GenericMap{
+				"insecure": true,
+			},
+		}
+
+		if isTracingEnabled(dest) {
+			currentConfig.Service.Pipelines["traces/quickwit"] = commonconf.Pipeline{
```

Comment:
A user can potentially add multiple quickwit destinations - in which case, the key `"traces/quickwit"` of the last one will override the configuration of the prior one.

While rare, in other destinations we use 
```suggestion
                         tracesPipelineName := "traces/quickwit-" + dest.Name
			currentConfig.Service.Pipelines[tracesPipelineName] = commonconf.Pipeline{
```

which will still work even in the present of multiple destinations of the same type.

Reply:
Done